Library/Formula/fakesmtp.rb

```diff
@@ -0,0 +1,53 @@
+class Fakesmtp < Formula
+  desc "Dummy SMTP server with GUI for testing emails in applications easily"
+  homepage "https://nilhcem.github.io/FakeSMTP/"
+  url "https://github.com/Nilhcem/FakeSMTP/archive/v2.0.tar.gz"
+  sha256 "8cb623f32b55f814af43b731432eb14da80d95cb13059c79423265037cda9dfb"
+
+  depends_on :java
+  depends_on "maven" => :build
+
+  def install
+    ENV.java_cache
+    system "mvn", "package", "-Dmaven.test.skip=true"
+    libexec.install "target/fakeSMTP-#{version}.jar"
+    (bin/"fakesmtp").write <<-EOS.undent
```

Comment:
@mikemcquaid I add the java flag `-Djava.awt.headless=true` to prevent the splashscreen if started in the background (-b or --background).

![screen shot 2016-03-19 at 00 10 04](https://cloud.githubusercontent.com/assets/423234/13894339/f9a86f76-ed66-11e5-8b26-679be411e8de.png)

Are you sure you want me to remove this?